Makefile

```diff
@@ -285,7 +285,7 @@ jupytext--to-ipynb: ## convert remote md files into ipynb
 
 reset-db: ## Reset the PostgreSQL database
 	$(COMPOSE) stop
-	$(COMPOSE) down postgresql metabase
+	# $(COMPOSE) down postgresql metabase
```

Comment:
dead code or forgot to uncomment?

Reply:
This is something I need to fix separately.